src/Splitio/Domain/EventsManagerConfig.cs

```diff
@@ -62,13 +62,15 @@ public static EventsManagerConfig BuildEventsManagerConfig()
                     }
                 }
             };
+
             Dictionary<SdkEvent, HashSet<SdkInternalEvent>> suppressedBy = new Dictionary<SdkEvent, HashSet<SdkInternalEvent>>
```

Comment:
I didn't see this earlier, but this should be 
```suggestion
            Dictionary<SdkEvent, HashSet<SdkEvent>> suppressedBy = new Dictionary<SdkEvent, HashSet<SdkEvent>>
```

External events are suppressed by external events.